python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -61,6 +61,7 @@ def execute(self, context: Context):
             parameters=self.parameters,
         )
         context["ti"].xcom_push(key="output_table_row_count", value=str(self.output_table.row_count))
+        context["ti"].xcom_push(key="output_table_conn_id", value=str(self.output_table.conn_id))
```

Comment:
We shall add a comments here to remove it in future

Reply:
+1

Reply:
Added the comment